libs/server/shared/src/services/market-data.service.ts

```diff
@@ -303,25 +388,27 @@ export class PolygonMarketDataService implements IMarketDataService {
             exchangeName: string
         })[]
     > {
-        const shouldRateLimit = process.env.NX_POLYGON_TIER === 'basic'
-        const exchanges = await this.api.reference.exchanges({
+        const allExchanges = await this.api.reference.exchanges({
             locale: 'us',
             asset_class: 'stocks',
         })
 
+        // Only get tickers for exchanges, not TRF or SIP
+        const exchanges = allExchanges.results.filter((exchange) => exchange.type === 'exchange')
```

Comment:
@karan0handa updated to filter out TRFs and SIPs, should make the sync on basic tier a little less slow